### PR TITLE
Fix dnu list regression

### PR DESCRIPTION
**Pull request by troydai**
https://github.com/aspnet/dnx/issues/1637

### CODE REVIEW

File: src/Microsoft.Framework.PackageManager/List/LibraryDependencyFlatRenderer.cs [hunk 1]
```diff
@@ -89,7 +89,7 @@ public IEnumerable<string> GetRenderContent(IGraphNode<LibraryDescription> root)
 
                 if (!_hideDependent)
                 {
-                    var dependents = string.Join(", ", dependenciesMap[description].Select(dep => dep.ToString()).OrderBy(name => name));
+                    var dependents = string.Join(", ", dependenciesMap[description].Select(dep => dep.Identity.ToString()).OrderBy(name => name));
```
Comment (davidfowl) [inline src/Microsoft.Framework.PackageManager/List/LibraryDependencyFlatRenderer.cs]: Identity can obviously never be null right?

Comment (troydai) [inline src/Microsoft.Framework.PackageManager/List/LibraryDependencyFlatRenderer.cs]: Not here. If you track down the source of the LibraryDescriptions' source, their Identities are already checked before the tree is created. *If* the identity is null, the failure will happen in the very beginning of dnu list.

Comment (davidfowl) [inline src/Microsoft.Framework.PackageManager/List/LibraryDependencyFlatRenderer.cs]: Can you write a test for this?

Comment (troydai) [inline src/Microsoft.Framework.PackageManager/List/LibraryDependencyFlatRenderer.cs]: Yup. But lets finish the analyzing first. The original source of all the LibraryDescriptions is ApplicaitonHostContext.DependencyWalker https://github.com/aspnet/dnx/blob/dev/src/Microsoft.Framework.PackageManager/List/DependencyListOperation.cs#L31. Any chance it returns any LibraryDescription with null Identity?

**Comment by pranavkm** [comment]
:shipit: !

**Pull request merged**